<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hitungan Bowditch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="radio_pTertutup">
       <property name="text">
        <string>Poligon Tertutup</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_pTerbuka">
       <property name="text">
        <string>Poligon Terbuka</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Titik Awal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="last_point"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Koordinat Titik Awal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_last_point">
        <property name="text">
         <string>Nama Titik Akhir</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="y_titik_awal"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x_titik_awal"/>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_pilihKoord">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Pillih dari peta</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Daftar Titik</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Nama Titik</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Jarak</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Azimuth</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="btn_tambahTitik">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Tambah Titik</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_hapusTitik">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Hapus Titik</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_hitungKoord">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Hitung Koordinat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_bowditch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Koreksi Bowditch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_resetHitung">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Reset Hitung</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_hapusTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Hapus Daftar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>257</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_importTitik">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Buka Daftar Titik</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_exportTitik">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Simpan Daftar Titik</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_plotTitik">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Plot titik</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radio_pTertutup</sender>
   <signal>pressed()</signal>
   <receiver>label_last_point</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_pTertutup</sender>
   <signal>pressed()</signal>
   <receiver>last_point</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_pTerbuka</sender>
   <signal>pressed()</signal>
   <receiver>last_point</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_pTerbuka</sender>
   <signal>pressed()</signal>
   <receiver>label_last_point</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
